Dockerfile

```diff
@@ -17,21 +17,21 @@
 #-------------------------------------------------------------------------------
 # Pre-requisites required before using this Dockerfile:
 #   1. An installed and working Docker/Moby
-#   2. Register for and download Actian Vector Community Edition
-#   3. Copy the Vector *.tgz file you downloaded to the same location as this Dockerfile
+#   2. Get *.tgz file of licensed Vector or register for and download Actian Vector Community Edition
```

Comment:
I think Actian Vector Community Edition is going away? Recommend just useing the "Get *.tgz file of licensed Vector or register" piece

Reply:
aww crap! Thanks for catching this.

@auswells  and @monda07  any reason to not merge #11? We need to figure that out, 'cos someone will need to marge and the question is who. @monda07 change doesn't remove the refs that @fgropengieser changes makes (that that we definitely want).

Reply:
Well, I COMPLETELY forgot about my request #11 !  
The proper technique is to merge one pull request, merge master into the other branch and merge that pull request.  Feel free to merge this one and I'll fix up my pull request and we can finish

Reply:
Hi.

Regarding the Daryl's Pull-Request: My question would be, should we still build on Docker Compose at this time? My feeling is, that this is kind of obsolete due to the popularity of Kubernetes and Helm.

Regarding Chris's comment: What do you mean with Community Edition is going away? I  am not fully aware of what is going on there... My understand was, that we just wanted to extend the docker file to also support licensed Vector!?

Reply:
Yes, the Compose example should remain because not all environments need the massive complexity of a full Kubernetes environment.  The primary idea here is to provide to the end user a variety of different implementation styles so that they can select the one that suits their needs.  "docker run" "docker compose" "docker stack" and Kubernetes are all valid deployment strategies.

I would think that _adding_ a Helm chart would be an excellent idea (but I'm not doing it ... ;-)

Reply:
Off topic for this PR now we've agreed to get this in first ;)

@fgropengieser please could you make the final tweak to the comment and we can then get this merged :)

Reply:
Done.

Reply:
👍  thanks!